CIFAR/main.swift

```diff
@@ -0,0 +1,44 @@
+import TensorFlow
+
+let batchSize: Int32 = 4
+
+let (trainingDataset, testDataset) = loadCIFAR10()
+let trainingBatches = trainingDataset.batched(Int64(batchSize))
+let testBatches = testDataset.batched(Int64(batchSize))
+
+let modeRef = ModeRef()
+var net = CIFARNet() // modeRef: modeRef)
```

Comment:
nit: rename for consistency?
```suggestion
var model = CIFARNet() // modeRef: modeRef)
```